From b9b51f556c8223c82940c55834987e2e2b21d426 Mon Sep 17 00:00:00 2001
From: thongsyho <thong.ho.px@rvc.renesas.com>
Date: Wed, 1 Nov 2017 15:11:34 +0700
Subject: [PATCH 201/641] ARM: shmobile: r8a77470: Add clock index macros for
 DT sources

Add macros usable by device tree sources to reference r8a77470 clocks by index.

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
---
 include/dt-bindings/clock/r8a77470-clock.h | 147 +++++++++++++++++++++++++++++
 1 file changed, 147 insertions(+)
 create mode 100644 include/dt-bindings/clock/r8a77470-clock.h

diff --git a/include/dt-bindings/clock/r8a77470-clock.h b/include/dt-bindings/clock/r8a77470-clock.h
new file mode 100644
index 0000000..c40c9d7
--- /dev/null
+++ b/include/dt-bindings/clock/r8a77470-clock.h
@@ -0,0 +1,147 @@
+/*
+ * Copyright (C) 2016 Renesas Electronics Corporation
+ * Copyright 2013 Ideas On Board SPRL
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ */
+
+#ifndef __DT_BINDINGS_CLOCK_R8A77470_H__
+#define __DT_BINDINGS_CLOCK_R8A77470_H__
+
+/* CPG */
+#define R8A77470_CLK_MAIN		0
+#define R8A77470_CLK_PLL0		1
+#define R8A77470_CLK_PLL1		2
+#define R8A77470_CLK_PLL3		3
+#define R8A77470_CLK_LB			4
+#define R8A77470_CLK_QSPI		5
+#define R8A77470_CLK_SDH		6
+#define R8A77470_CLK_SD0		7
+#define R8A77470_CLK_SD1		8
+#define R8A77470_CLK_Z			9
+
+/* MSTP0 */
+#define R8A77470_CLK_MSIOF0		0
+#define R8A77470_CLK_HUDI		18
+#define R8A77470_CLK_INTCRT		22
+
+/* MSTP1 */
+#define R8A77470_CLK_VCP0		1
+#define R8A77470_CLK_VPC0		3
+#define R8A77470_CLK_TSIF1		7
+#define R8A77470_CLK_TSIF0		8
+#define R8A77470_CLK_TMU1		11
+#define R8A77470_CLK_PVRSRVKM		12
+#define R8A77470_CLK_2DDMAC		15
+#define R8A77470_CLK_FDP0		19
+#define R8A77470_CLK_TMU3		21
+#define R8A77470_CLK_TMU2		22
+#define R8A77470_CLK_CMT0		24
+#define R8A77470_CLK_VSP1_DU0		28
+#define R8A77470_CLK_VSP1_S		31
+
+/* MSTP2 */
+#define R8A77470_CLK_MSIOF2		5
+#define R8A77470_CLK_MSIOF1		8
+#define R8A77470_CLK_MFIS		13
+#define R8A77470_CLK_SYS_DMAC1		18
+#define R8A77470_CLK_SYS_DMAC0		19
+#define R8A77470_CLK_SEC_ENGIN		28
+
+/* MSTP3 */
+#define R8A77470_CLK_SDHI2		12
+#define R8A77470_CLK_SDHI1		13
+#define R8A77470_CLK_SDHI0		14
+#define R8A77470_CLK_PCIE		19
+#define R8A77470_CLK_USBDMAC0_1		26
+#define R8A77470_CLK_USBDMAC1_1		27
+#define R8A77470_CLK_CMT1		29
+#define R8A77470_CLK_USBDMAC0_0		30
+#define R8A77470_CLK_USBDMAC1_0		31
+
+/* MSTP4 */
+#define R8A77470_CLK_RWDT		2
+#define R8A77470_CLK_USBDDM0		6
+#define R8A77470_CLK_IRQC		7
+#define R8A77470_CLK_INTSYS		8
+#define R8A77470_CLK_USBDDM1		9
+
+/* MSTP5 */
+#define R8A77470_CLK_AUDIO_DMAC0	2
+#define R8A77470_CLK_PWM		23
+#define R8A77470_CLK_BROM		30
+
+/* MSTP7 */
+#define R8A77470_CLK_EHCI0		3
+#define R8A77470_CLK_HSUSB0		4
+#define R8A77470_CLK_EHCI1		5
+#define R8A77470_CLK_HSUSB1		6
+#define R8A77470_CLK_HSCIF2		13
+#define R8A77470_CLK_SCIF5		14
+#define R8A77470_CLK_SCIF4		15
+#define R8A77470_CLK_HSCIF1		16
+#define R8A77470_CLK_HSCIF0		17
+#define R8A77470_CLK_SCIF3		18
+#define R8A77470_CLK_SCIF2		19
+#define R8A77470_CLK_SCIF1		20
+#define R8A77470_CLK_SCIF0		21
+#define R8A77470_CLK_DU1		23
+#define R8A77470_CLK_DU0		24
+#define R8A77470_CLK_LVDS0		26
+#define R8A77470_CLK_DVENC		27
+
+/* MSTP8 */
+#define R8A77470_CLK_IPMMU_GP		0
+#define R8A77470_CLK_DVDEC		1
+#define R8A77470_CLK_VIN1		10
+#define R8A77470_CLK_VIN0		11
+#define R8A77470_CLK_AVB		12
+#define R8A77470_CLK_ETHER		13
+
+/* MSTP9 */
+#define R8A77470_CLK_IRR		3
+#define R8A77470_CLK_GPIO5		7
+#define R8A77470_CLK_GPIO4		8
+#define R8A77470_CLK_GPIO3		9
+#define R8A77470_CLK_GPIO2		10
+#define R8A77470_CLK_GPIO1		11
+#define R8A77470_CLK_GPIO0		12
+#define R8A77470_CLK_RCAN0		15
+#define R8A77470_CLK_RCAN1		16
+#define R8A77470_CLK_QSPI_MOD1		17
+#define R8A77470_CLK_QSPI_MOD0		18
+#define R8A77470_CLK_ADG		22
+#define R8A77470_CLK_I2C4		27
+#define R8A77470_CLK_I2C3		28
+#define R8A77470_CLK_I2C2		29
+#define R8A77470_CLK_I2C1		30
+#define R8A77470_CLK_I2C0		31
+
+/* MSTP10 */
+#define R8A77470_CLK_SSI		5
+#define R8A77470_CLK_SSI9		6
+#define R8A77470_CLK_SSI8		7
+#define R8A77470_CLK_SSI7		8
+#define R8A77470_CLK_SSI6		9
+#define R8A77470_CLK_SSI5		10
+#define R8A77470_CLK_SSI4		11
+#define R8A77470_CLK_SSI3		12
+#define R8A77470_CLK_SSI2		13
+#define R8A77470_CLK_SSI1		14
+#define R8A77470_CLK_SSI0		15
+#define R8A77470_CLK_SCU		17
+#define R8A77470_CLK_DVC1		18
+#define R8A77470_CLK_DVC0		19
+#define R8A77470_CLK_CTU0		20
+#define R8A77470_CLK_CTU1		21
+#define R8A77470_CLK_SRC6		25
+#define R8A77470_CLK_SRC5		26
+#define R8A77470_CLK_SRC4		27
+#define R8A77470_CLK_SRC3		28
+#define R8A77470_CLK_SRC2		29
+#define R8A77470_CLK_SRC1		30
+
+#endif /* __DT_BINDINGS_CLOCK_R8A77470_H__ */
-- 
2.7.4

